src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
Extra empty line (we do not put an empty line between parent and child elements)
```suggestion
    <header class="header">
      <div class="header__logo"></div>
```